com.unity.shadergraph/Editor/Importers/ShaderGraphImporterEditor.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using UnityEditor.AssetImporters;
+using UnityEditor.Callbacks;
+using UnityEditor.GraphToolsFoundation.Overdrive;
+using UnityEditor.ShaderGraph.GraphUI;
+using UnityEngine;
+
+namespace UnityEditor.ShaderGraph
+{
+    [CustomEditor(typeof(NewShaderGraphImporter))]
+    class NewShaderGraphImporterEditor : ScriptedImporterEditor
+    {
+        MaterialEditor materialEditor = null;
+        public override void OnInspectorGUI()
+        {
+            EditorGUILayout.HelpBox("WIP", MessageType.Info);
```

Comment:
nice touch :D

Reply:
Lol wanted to make sure the place to work on the code was seen